pygmt/datasets/samples.py

```diff
@@ -30,6 +30,7 @@ def list_sample_data():
         "mars_shape": "Table of topographic signature of the hemispheric dichotomy of "
         " Mars from Smith and Zuber (1996)",
         "ocean_ridge_points": "Table of ocean ridge points for the entire world",
+        "table511": "Table 5.11 in Davis: Statistics and Data Analysis in Geology.",
```

Comment:
I don't think `table511` is a good name for this dataset, but I have no idea about the content of the dataset.

Reply:
The main reason I went with `table511` is because that is in line with the GMT remote dataset name. I also have no idea what the content is.

Reply:
@PaulWessel Could you please explain what the `Table_5_11.txt` file has?

Reply:
Yikes.  My book is sitting on a shelf in Hawaii. I could answer in 12 days when I am back for a few weeks.  But vague memory is that is it topography.

Reply:
This dataset is used in ex12 (https://docs.generic-mapping-tools.org/latest/gallery/ex12.html). 

> Our next example operates on a data set of topographic readings non-uniformly distributed in the plane (Table 5.11 in Davis: Statistics and Data Analysis in Geology, J. Wiley). 

According to the example description, perhaps we can name the dataset as "unevenly_sampled_topography"?

Reply:
Went to my library and found the book, but it's the Third edition from 2002 so the numbering is different 
![IMG_20220511_111058__01.jpg](https://user-images.githubusercontent.com/23487320/167886793-dd829686-b7a8-42d7-a695-b6e975352593.jpg)
However, I managed to find the contour map similar to ex12 at page 379

![IMG_20220511_111213__01.jpg](https://user-images.githubusercontent.com/23487320/167887090-44512f41-191e-48b8-b386-f9c16ea1ab76.jpg)

So it seems to be ground control points of topography over Notredam? Here's a few more pictures

![IMG_20220511_111455.jpg](https://user-images.githubusercontent.com/23487320/167887390-d65b6f65-9cd4-4ae9-be18-960623930a65.jpg)
![IMG_20220511_111529__01.jpg](https://user-images.githubusercontent.com/23487320/167887423-3a749f07-97db-4900-832a-9445a16078e6.jpg)

So the first column is X, second column is Y, and third column is Elevation in feet above sea level.

Could call it `notredam_topography` or `notredam_ground_control_points`🙂

Reply:
Oh, and I found the same `NOTREDAM.txt` file at https://github.com/EASC-305/EASC-305-Spring-2020/blob/bfe22e94fbcd0ed37d5b8c5a4863c12bc2cc414c/Lec_Demos/07_SpatialAnalysis/NOTREDAM.TXT and https://github.com/XiangyunHuang/Thesis-Template-Bookdown/blob/e822f9ee166179a53e83ec28664dfc70c4d70e4c/data/NOTREDAM.TXT. 

Looking at a map of Paris though, I'm not sure if this is the Notre-Dame I'm thinking of since the streams don't seem to align. Maybe there's another place called Notredam?

Reply:
Given John Davis worked for the Kansas Geological Survey I am guessing it is either in Kansas or this is Notre Dame University area in Indiana.

Reply:
Would `notre_dame_topgraphy` work?

Reply:
> Would `notre_dame_topgraphy` work?

Sounds a good name to me.

Reply:
Presumably meant notre_dame_topography ?

Reply:
Yes! The dangers of getting on my computer pre-coffee.